app/assets/javascripts/temp_pax_options.js

```diff
@@ -0,0 +1,18 @@
+$(document).on('turbolinks:load', function(){
+  $('#passenger_permanent').change(function(){
+    var registeredbyField = $('#pax_registered_with_DS');
+    var hasbrochureField = $('#pax_has_brochure');
+    if($(this).is(':checked')){
+      registeredbyField.prop('disabled', true);
+      registeredbyField.prop('checked', false);
+    }
+    else registeredbyField.prop('disabled', false)
+         registeredbyField.prop('checked', false)
```

Comment:
I'm not too sure about the JS here, namely:

```js
else something();
somethingElse();
```

whether both `something`s are actually attached to the `else`.

Reply:
It's functional ... I'm checking into whether or not there is a better way to do it. Basically, in the `else` case, I want that particular checkbox to be enabled and unchecked. I guess since it un-checks when the box is disabled, what I'm doing here might actually be redundant since it should be unchecked initially, anyways.

Reply:
No, I don't think that syntax is right.  The `'checked'` line is **outside** the else block. It doesn't feel like that what you intended. You probably want to add some `{}`s

Example:   https://jsbin.com/xoqudafaqu/edit?html,js,output